training/tests/integration/conftest.py

```diff
@@ -222,3 +223,18 @@ def gnn_config_with_data(
     cfg = OmegaConf.merge(template, testing_modifications_with_temp_dir, use_case_modifications)
     OmegaConf.resolve(cfg)
     return cfg
+
+
+@pytest.fixture
+def gnn_config_with_checkpoint(gnn_config_with_data: OmegaConf) -> OmegaConf:
+    existing_ckpt = get_test_data("anemoi-integration-tests/training/checkpoints/testing-checkpoint-global-Jun-24.ckpt")
```

Comment:
Would this filename enable versioning it? I would imagine that in the future, a new checkpoint would be needed so having some kind of version number in the path could help to keep things sorted. E.g. one could includethe version number of anemoi-training that generated this checkpoint.

And out of curiosity, how is this checkpoint been generated? Is it simply the result of `test_training_cycle_without_config_validation`?

Reply:
I would be fine with using the creation date. But I would suggest including the year and perhaps a numerical representation of the month. e.g. `testing-checkpoint-global-2025-06-24.ckpt`

Reply:
I was going to propose symlinks, but these don't seem to be supported by S3

Reply:
It's probably best to make it clear in the test then, and use the dated name as you have above. It means more code changes, but its more transparent.